docs/reference/commandline/swarm_join_token.md

```diff
@@ -11,7 +11,7 @@ parent = "smn_cli"
 # swarm join-token
 
 ```markdown
-Usage:	docker swarm join-token [--rotate] (worker|manager)
+Usage:	docker swarm join-token [OPTIONS] (worker|manager)
```

Comment:
Yes, I was in doubt; I'm +1 for consistency, but also kinda liked the `[-q] [--rotate]` here 😇